### ISSUE TITLE
 While closing the viewcontroller via dragging, calling floatPanelController's hide() will cause a crash.

### ISSUE THREAD

**Issue by cozzin** [SCENEE/FloatingPanel#216]
### Short description
While closing the viewcontroller via dragging, calling floatPanelController's hide() will cause a crash.

### Expected behavior
1. Drag to close view
2. Touch backdropview (call hide() method) while dragging to close view controller
3. The hide () command is ignored because the screen is already closed

### Actual behavior
1. Drag to close view
2. Touch backdropview (call hide() method) while dragging to close view controller
3. Crash [image](https://user-images.githubusercontent.com/11647461/58791060-7c15d780-862c-11e9-8086-73a72666c2af.png)

### Steps to reproduce
1. Add gesture for backdrop
2. Close Panel using drag
3. Touch backdrop view while the screen is closing

**Code example that reproduces the issue**
```swift
public final class ExampleViewController: FloatingPanelController {

    private lazy var backdropTapGesture = UITapGestureRecognizer(target: self, action: #selector(handleBackdrop(_:)))

    init() {
        super.init()
        backdropView.addGestureRecognizer(backdropTapGesture)
    }

    @objc func handleBackdrop(_ sender: UITapGestureRecognizer) {
        hide()
    }
}
```

### Environment

**Library version**
1.5.1

**Installation method**

- [x] CocoaPods
- [ ] Carthage
- [ ] Git submodules

**iOS version(s)**
12.2

**Xcode version**
10.2.1

**Comment by SCENEE** [comment]
Thanks, @cozzin! I will fix it.

**Comment by SCENEE** [comment]
I fixed #220. Thanks again, @cozzin!

**Comment by cozzin** [comment]
@SCENEE You're welcome. I am a big fan of this library. Thank you for your quick fix.
